app/assets/stylesheets/application.css.scss

```diff
@@ -32,6 +32,13 @@ body {
   background-position: center center;
 }
 
+.application-body-dead{
+  background-image: url('dead.jpg');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space